internals/reminders.py

```diff
@@ -100,7 +100,7 @@ def build_email_tasks(
     subject = subject_format % fe.name
     if is_escalated:
       if EMAIL_SUBJECT_PREFIX in subject:
-        subject = subject.replace(EMAIL_SUBJECT_PREFIX, 'Escalation request')
+        subject = subject.replace(EMAIL_SUBJECT_PREFIX, '[Escalated Request]')
```

Comment:
Nit: Since we are changing this, it would be a good time to change this to a constant. ESCALATION_EMAIL_SUBJECT_PREFIX